_docs-sources/library/principles/be-judicious-with-new-features.md

```diff
@@ -0,0 +1,24 @@
+# Be Judicious With New Features
+
+Sometimes new features in OpenTofu are released that make module authoring more convenient.
+
+Leveraging them can make it convenient author modules, but it can be really inconvenient for module consumers, as they can be inadvertently forced into adopting the newer version of OpenTofu that supports those new features.
```

Comment:
Or worse, they can't use the library (or recent versions of the library) because they're UNABLE to update their version of tofu